lsmtool/tableio.py

```diff
@@ -674,7 +674,7 @@ def rowStr(row, metaDict):
                     dstr = ' '
                 else:
                     dstr = fillVal
-        elif str(d) == '-9999':
+        elif str(d)[:5] == '-9999':
```

Comment:
You could also use `str(d).startswith('-9999')`. On a related note: what is this magic `-9999` value?

Reply:
Ah, OK. I was just curious. So `-9999` is a value that can never occur in real life, I guess. It's really a kind of `None` value, right?

Reply:
Yes, that's right. Usually I would use `None` in such cases, but I guess it didn't work for some reason.